clusterman/aws/client.py

```diff
@@ -74,11 +75,21 @@ def _init_session():
     global _session
 
     if not _session:
-        _session = boto3.session.Session(
-            staticconf.read_string("accessKeyId", namespace=CREDENTIALS_NAMESPACE),
-            staticconf.read_string("secretAccessKey", namespace=CREDENTIALS_NAMESPACE),
-            region_name=staticconf.read_string("aws.region"),
-        )
+        # when running clusterman's CLI, we want to use folks' personal AWS creds
+        # so that we don't need to distribute service user creds (and since we can't
+        # use Pod Identity outside of k8s :p)
+        if os.getenv("AWS_PROFILE"):
+            if os.getenv("AWS_REGION") is None:
+                raise InvalidConfigurationError("Setting AWS_REGION is required if setting a specific AWS profile")
+            _session = boto3.session.Session(
+                region_name=os.getenv("AWS_REGION"),
```

Comment:
Hmm, interesting that we set this at the session level...I thought technically we can set this later when we generate the individual clients, and at that point we _should_ have access to the cluster config's `aws_region` setting.
But not a blocker at all, I know we can set this in the wrappers (or yelsoa-configs for instances) to sidestep!

Reply:
fixed :)